.circleci/config.yml

```diff
@@ -649,6 +649,43 @@ jobs:
           command: |
             .circleci/torchscript_bc_test/setup_master_envs.sh
             .circleci/torchscript_bc_test/validate_objects.sh
+  build_docs:
+    <<: *binary_common
+    docker:
+      - image: "pytorch/manylinux-cuda100"
+    resource_class: 2xlarge+
+    steps:
+      - attach_workspace:
+          at: ~/workspace
+      - run:
```

Comment:
@mattip Looks like `- checkout` step is missing, so the repository is not fetched, thus `packaging` directory is not found.

Also for the completeness, can you add `- designate_upload_channel` too?